### PR TITLE
fix(trim query before search)

### LINKED ISSUE #1364: Trim query before search
#### Nature of issue?
- Existing feature enhancement

- Operating System: Ubuntu 18.04

#### Feature enhancement details:
Search Bar in the modal should only search for valid query, not for empty spaces.

### PR DESCRIPTION
**Pull request by shakti97**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

Fixes #1364

### CODE REVIEW

File: client/modules/IDE/components/Searchbar/Searchbar.jsx [hunk 1]
```diff
@@ -26,11 +26,12 @@ class Searchbar extends React.Component {
...
     if (e.key === 'Enter') {
-      this.props.setSearchTerm(this.state.searchValue);
+      this.searchChange();
     }
   }
 
-  searchChange = (value) => {
+  searchChange = () => {
+    if (this.state.searchValue.trim().length === 0) return;
     this.props.setSearchTerm(this.state.searchValue);
```
Comment (catarak) [inline client/modules/IDE/components/Searchbar/Searchbar.jsx:35]: Shouldn't you be using `this.state.searchValue.trim()` as the input value here?

Comment (shakti97) [reply client/modules/IDE/components/Searchbar/Searchbar.jsx:35]: yes, will do the recommended changes

**Comment by totalcloud-tc** [comment]
@catarak Added the recommended changes , can you please review and suggested changes :)

**Comment by catarak** [comment]
thank you!

**Pull request merged**